src/Hint/Extensions.hs

```diff
@@ -425,7 +427,11 @@ used EmptyCase = hasS f
     f _ = False
 used KindSignatures = hasT (un :: HsKind GhcPs)
 used BangPatterns = hasS isPBangPat ||^ hasS isStrictMatch
-used TemplateHaskell = hasS $ not . isQuasiQuoteSplice
+used TemplateHaskell = hasS (not . isQuasiQuoteSplice) ||^ hasS isTypedSplice
+  where
+    isTypedSplice :: HsExpr GhcPs -> Bool
```

Comment:
please rebase past 8c3cc3eb2a329cc26d34a3525fe944767bbe1d0a and use `Language.Haskell.Syntax.Expr.isTypedSplice`.

Reply:
Done. Thanks for including this in the lib :smiley: